scripts/cc-cli/src/commands/collectCoins.ts

```diff
@@ -31,9 +45,12 @@ async function collectCoinsAction(options: OptionValues) {
   } = await newApi(options.url);
   const signer = await initCallerKeyring(options);
 
-  requestCollectCoins(options.externalAddress, signer, options.burnTxHash)
-    .then(handleSuccess)
-    .catch(handleError);
+  const event = await requestCollectCoins(
+    options.externalAddress,
+    signer,
+    options.burnTxHash,
+  );
+  await event.waitForVerification(800_000);
```

Comment:
```suggestion
  await event.waitForVerification(900_000);
```

We currently guarantee that a given collect coins will be dealt with in 60 blocks (after that the task expires and you'd have to re-submit the transaction). So in the worst case `60 * 15s = 900_000`

Reply:
This change and the others mentioned are addressed in #1326. Please keep the conversation unresolved for easier reference :)